AzureSignTool/AuthenticodeKeyVaultSigner.cs

```diff
@@ -103,7 +106,6 @@ public void Dispose()
         {
             _chain.Dispose();
             _certificateStore.Close();
-            _configuration.Dispose();
```

Comment:
This was actually a bug. We shouldn't be disposing of an object we don't own since it's passed in through the constructor.